### PR TITLE
First attempt at resolving the NRE in single player mode

### PR DESCRIPTION
**Pull request by sbtoonz**
when it canot load the zpkg from server because server does not exist in this situation

### CODE REVIEW

File: EpicLoot/Adventure/BountyTarget.cs [hunk 1]
```diff
@@ -61,11 +61,13 @@ private void OnDeath()
         public void Initialize(BountyInfo bounty, string monsterID, bool isAdd)
         {
             _zdo.Set(BountyIDKey, bounty.ID);
-
-            var pkg = new ZPackage();
-            bounty.ToPackage(pkg);
-            pkg.SetPos(0);
-            _zdo.Set(BountyDataKey, pkg.GetBase64());
+            if (ZNet.instance.IsServer() && ZNet.instance.IsDedicated() || !ZNet.instance.IsServer() && !ZNet.instance.IsDedicated())
```
Comment (RandyKnapp) [inline EpicLoot/Adventure/BountyTarget.cs:64]: What is this actually doing? The operator precedence here means this does something that you probably didn't mean to do?

**Comment by sbtoonz** [comment]
Checks if game is playing in only local world instance vs client to server instance 
goal was to run the RPC call only when playing as client 

I use this same logic check to bypass admin RPC check in my DevFX mod in local play instances

**Comment by sbtoonz** [comment]
so basically is saying 

If znet is local OR if znet is client 
then do the Rpc call

**Comment by sbtoonz** [comment]
I do need someone on your team to validate the contents of this if statement and see if it could be simplified 

essentially this PR is in reference to issue #277

**Comment by RandyKnapp** [comment]
I think I see what you're doing here. I will get this in today.

**Comment by sbtoonz** [comment]
Znet.instance.IsServer() && !Znet.instance.IsDedicated()    <--- local or local world play 

!Znet.instance.IsServer() && !Znet.instance.IsDedicated() <-- local started client connected to a server

Znet.instance.IsServer() && Znet.instance.IsDedicated() <---- dedicated server

**Pull request merged**